chainer/cuda.py

```diff
@@ -36,9 +36,17 @@
 import os
 
 import numpy
+import pkg_resources
 import six
 
 try:
+    pkg_resources.require([
+        'pycuda>=2014.1',
+        'scikits.cuda>=0.5.0b2,!=0.042',
+        'Mako',
+        'six>=1.9.0',
+    ])
```

Comment:
The list of dependent modules and their versions is duplicated to one in `cuda_deps/setup.py`. Can you get the list from `setup.py` to avoid the difference between them?